public/restaurant-style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
+*:focus {
+  outline: none !important;
+}
+
+body {
+  display: grid;
+  width: auto;
+  height: 100vh;
+  grid-template-rows: 5rem auto 5rem;
+
+}
+/*Header //////////////////////////////////////////////////////////*/
+#header {
+  display: grid;
+  grid-template-columns: 10.5rem 10.5rem auto;
+  background-color: #87f5ff;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1600 900'%3E%3Cpolygon fill='%23547ccc' points='957 450 539 900 1396 900'/%3E%3Cpolygon fill='%2361cdff' points='957 450 872.9 900 1396 900'/%3E%3Cpolygon fill='%23566fbf' points='-60 900 398 662 816 900'/%3E%3Cpolygon fill='%2301c4e2' points='337 900 398 662 816 900'/%3E%3Cpolygon fill='%235861b2' points='1203 546 1552 900 876 900'/%3E%3Cpolygon fill='%2300b8bc' points='1203 546 1552 900 1162 900'/%3E%3Cpolygon fill='%235954a4' points='641 695 886 900 367 900'/%3E%3Cpolygon fill='%2300aa90' points='587 900 641 695 886 900'/%3E%3Cpolygon fill='%23594796' points='1710 900 1401 632 1096 900'/%3E%3Cpolygon fill='%23379a63' points='1710 900 1401 632 1365 900'/%3E%3Cpolygon fill='%23593987' points='1210 900 971 687 725 900'/%3E%3Cpolygon fill='%23558838' points='943 900 1210 900 971 687'/%3E%3C/svg%3E");
+  background-attachment: fixed;
+  background-size: cover;
+
+}
```

Comment:
You could make a 3rd css file for general css styles which is included in both pages, so you wouldn't have to repeat the styles for these shared elements.